lib/platform-stats.js

```diff
@@ -335,17 +336,39 @@ export const refreshDatabase = async (
  * @param {import('pg').Client} pgClient
  */
 export const updateTopMeasurementParticipants = async (pgClient) => {
+  const mvTableName = 'top_measurement_participants_yesterday_mv'
   try {
     await pRetry(async () => {
-      await pgClient.query('REFRESH MATERIALIZED VIEW top_measurement_participants_yesterday_mv')
+      await pgClient.query(`REFRESH MATERIALIZED VIEW ${mvTableName}`)
     }, {
       retries: 3,
       onFailedAttempt: (error) => {
-        console.warn(`Attempt to refresh materialized view failed: ${error.message}. Retrying...`)
+        console.warn(`Attempt to refresh ${mvTableName} failed: ${error.message}. Retrying...`)
       }
     })
   } catch (err) {
-    console.error('Error refreshing top measurement participants', err)
+    console.error(`Error refreshing ${mvTableName}`, err)
+    Sentry.captureException(err)
+  }
+}
+
+/**
+ * @param {import('pg').Client} pgClient
+ */
+export const updateTopMeasurementParticipantsView = async (pgClient) => {
```

Comment:
If we were going to keep both the old schema and the new one, it would make sense to combine these two methods (since they're basically the exact same code)

However, we're planning to delete the old schema and its code soon, so I thought it might not be worth combining them (and then handling SQL query safeness with the inserted table name)

What do you guys think?

Reply:
I had it that way before, but then we just have to directly write the table name in a string 3 separate times. And the string is internal to the function, not a real parameter, so no risk that I see unless a future change turns it into a parameter, which would be possible anyway.

But we can remove it if preferred, no hard preference from my end